### PR TITLE
Added additional documentation for windows

### PR DESCRIPTION
**Pull request by cheog**
The documentation was about installing python on windows. Not sure if I'm doing this right.

### CODE REVIEW

File: docs/installing_python.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Installing Python On Pipenv Windows
+
+You can either install python from the Microsoft Store or from the python
+website. It is recommended to install the Microsoft Store as it is better
```
Comment (ben-nathanson) [inline docs/installing_python.md:4]: `It is recommended to install the Microsoft Store` -> 'It is recommended to install Python from the Microsoft Store'

File: docs/installing_python.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+integrated into the system.
+
+## Actually Installing Python
+
+#### Installing from the Microsoft Store
+Open up the microsoft store and use the search to find python. Below is a
```
Comment (ben-nathanson) [inline docs/installing_python.md:10]: Capitalize `Microsoft Store`

File: docs/installing_python.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+
+#### Installing from python.org
+Simply go to the [downloads page](https://www.python.org/downloads/) and pick
+the version of python you want.
+
+### Checking the install
+To see if python is working go to the command line (either Command Prompt or
+Powershell) and type in `python`. If you installed from python.org it may be the
+case that ypu have to use `python.exe` instead of `python`. Either way, you
```
Comment (ben-nathanson) [inline docs/installing_python.md:24]: `ypu` -> `you`

File: docs/installing_python.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+![](resources/python_cmd_line.png)
+
+
+## Setting Your Environment Variables
+
+### Background
+You can install a bunch of different programs with python. For instance, later
+on we will be installing a program called Pipenv. Python puts that program into
+a "Scripts" folder, however, by default windows doesn't know where that
```
Comment (ben-nathanson) [inline docs/installing_python.md:36]: Capitalize Windows

File: docs/installing_python.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+"Scripts" folder is. Windows has an environment variable called "Path" that is a
+list of folders. When you try to run a program like Pipenv windows will look
```
Comment (ben-nathanson) [inline docs/installing_python.md:38]: Capitalize Windows

File: docs/setuphelp.md [hunk 6]
```diff
@@ -59,12 +59,11 @@ $ git merge upstream/master
 
 ## Python and Pipenv
 
-### Installing Python
+### [Installing Python and Pipenv on Windows](installing_python.md)
+If you want to install python on windows [click here](installing_python.md). If
```
Comment (ben-nathanson) [inline docs/setuphelp.md:63]: Capitalize Windows

File: docs/installing_python.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
+# Installing Python On Pipenv Windows
```
Comment (ben-nathanson) [inline docs/installing_python.md:1]: `Installing Python On Pipenv Windows` -> `Installing Python on Windows With Pipenv`

**Review by ben-nathanson — CHANGES_REQUESTED** [review]
Spelling nitpicks aside, this looks great! If someone with a Windows machine could review this that would be great, but I trust you. 

Thanks!

Ben

**Comment by bonfirefan** [comment]
I'm going to merge this, and we can correct spelling in a future update.

**Pull request merged**